aten/src/ATen/native/TensorConversions.cpp

```diff
@@ -117,4 +117,19 @@ Tensor to_mkldnn_backward(const Tensor& grad, const Tensor& input_) {
   return grad.to_dense();
 }
 
+Tensor view_dtype(const Tensor& self, ScalarType dtype) {
+  if (self.scalar_type() == dtype) {
+    return self;
+  }
+  TORCH_CHECK(self.element_size() == c10::scalarTypeToTypeMeta(dtype).itemsize(),
+    "Viewing a tensor as a new dtype with different bytes is not supported.");
```

Comment:
"with different bytes" -> "with a different number of bytes per element"

The original is ambiguous, since "with different bytes" could refer to the byte values being different, for example.

Reply:
fixed